templates/accounts/email_reset_email.html

```diff
@@ -11,6 +11,7 @@
 
 <{% absurl 'accounts-email-reset-complete' uid token %}>
 
-Your username, in case you've forgotten: {{ user.username }}
+If you have not requested an e-mail reset, you can safely ignore this email.
```

Comment:
I'm not happy about saying "safely ignore this email" - e.g. if you click the link then the address changes, right?
This email goes to the new email address - if a mistake was made when entering it then it might go to someone else? Though I see with the checks you require a logged-in user and the user making the change to be the account that requested the change.
I'm trying to step through the security considerations of someone breaking into an account (leaked password?) and changing the email address.

Reply:
As you mention the changes will only be applied if user is logged in. We could change the message and tell user to contact support, but I'm not too worried about this.

Reply:
Updated message 2d67df808d7320b052d1b6c6af6c72983d9c033f